environment.yml

```diff
@@ -2,12 +2,18 @@ name: flood_adapt
 
 channels:
   - conda-forge
+  - numba
```

Comment:
Why do we need a numba channel?

Reply:
Reason is that noaa-coops uses numpy >= 1.24 which is supported by numba 0.57. The latest version of numba i.e. 0.57 is not yet available on conda-forge, but on numba